### PR TITLE
CAD-1192: Restore trace verbosity configuration

### PR DESCRIPTION
**Pull request by Jimbo4350**
The trace verbosity was hard-coded in a large recent refactor, this PR removes the hardcoded value and restores the ability to change the tracing verbosity via the config yaml file.

### CODE REVIEW

**Comment by Jimbo4350** [comment]
Waiting on #1192 to merge

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 1]
```diff
@@ -901,12 +908,17 @@ traceCounter logValueName aCounter tracer = do
 
 tracerOnOff
   :: Transformable Text IO a
-  => Bool -> LoggerName -> Trace IO Text -> Tracer IO a
-tracerOnOff False _ _ = nullTracer
-tracerOnOff True name trcer = annotateSeverity
-                                $ toLogObject' MinimalVerbosity
+  => Bool -> TracingVerbosity -> LoggerName -> Trace IO Text -> Tracer IO a
```
Comment (CodiePP) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:911]: :+1:

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 2]
```diff
@@ -902,10 +908,10 @@ traceCounter logValueName aCounter tracer = do
 
 tracerOnOff
```
Comment (deepfire) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:909]: Maybe the function should get another name, as it clearly no longer does exactly what it name implies?

Comment (Jimbo4350) [reply cardano-node/src/Cardano/Tracing/Tracers.hs:909]: Feel free to suggest alternative names :slightly_smiling_face:

Comment (CodiePP) [reply cardano-node/src/Cardano/Tracing/Tracers.hs:909]: that's only delaying this PR

**Comment by Jimbo4350** [comment]
bors merge

**Pull request merged**